src/Application/Components/Store/Store.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import StoreItem from "./StoreItem";
+
+const Store = React.createClass({
+  propTypes: {
+    assets: React.PropTypes.array.isRequired,
+    createOnClick: React.PropTypes.func.isRequired,
+  },
+
+  componentDidMount() {
+    this.props.assets[0].unlock();
+    this.forceUpdate();
+  },
+
+  render() {
+    let unlockedAssets = [];
+    for (const asset of this.props.assets) {
+      if (asset.unlocked) {
+        unlockedAssets.push(asset);
+      }
+    }
```

Comment:
I don't think it's ticket worthy, just change these lines 16-21 to 
```
const unlockedAssets = _.where(this.props.assets, { unlocked: true });
```
That is, unless we change `this.props.assets` to an `Immutable.List` which is a good thing to do, then there's other `ImmutableJS` specific syntax to do this